src/app/manager-dashboard/certifications/certifications-add.component.ts

```diff
@@ -49,6 +51,9 @@ export class CertificationsAddComponent implements OnInit, AfterViewChecked {
           this.certificateForm.patchValue(certification);
           this.certificateInfo._rev = certification._rev;
           this.courseIds = certification.courseIds || [];
+          // here find out all courses in the system. compare all courses ids with this.coursesIds
```

Comment:
CoursesComponent already fetches all of the course system so all courses are available in `this.coursesTable.dataTable.data`